docs/websockets.md

```diff
@@ -109,3 +109,18 @@ correctly updated.
 
 * `await websocket.send(message)`
 * `await websocket.receive()`
+
+### Send Denial Response
+
+If you call `websocket.close()` before calling `websocket.accept()` then
+the server will automatically send a HTTP 403 error to the client.
+
+If you want to send a different error response, you can use the
+`websocket.send_denial_response()` method. This will send the response
+and then close the connection.
+
+* `await websocket.send_denial_response(response)`
+
+This requires the ASGI server to support the `websocket.http.response`
+extension. If it is not supported a `RuntimeError` will be raised and
+the application can then elect to perform a `websocket.close()` instead.
```

Comment:
I don't think informing about `RuntimeError` makes sense. The error raised on the code should contain the information needed for the user to take action.
```suggestion

```

Reply:
It helps the developer write the necessary boilerplate (a check, or an exception handler) without having to first experience the failure if his application is ever deployed with a server that does not offer the extension.